src/http.js

```diff
@@ -1,33 +1,73 @@
+import * as Cookie from './cookie';
+
+// consts
+import API_HOST from './api_host';
+import SDK_VERSION from './version';
 import STATUS from './status_codes';
 
-export function request(method, url, body, headers, successCallback, errorCallback) {
+function _handleResponse(response, jsonKey) {
+  if (jsonKey) {
+    const res = JSON.parse(response);
+    return res[jsonKey];
+  }
+  return response;
+}
+
+export function request(method, path, data, jsonKey, callback) {
```

Comment:
added a `jsonKey` parameter to grab the nested data in `_handleResponse` above

Reply:
Rather than passing the json key, could this just be handled in the callback?